cdap-hbase-compat-1.2-cdh5.7.0/pom.xml

```diff
@@ -0,0 +1,355 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright © 2015 Cask Data, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  use this file except in compliance with the License. You may obtain a copy of
+  the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  License for the specific language governing permissions and limitations under
+  the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>cdap</artifactId>
+    <groupId>co.cask.cdap</groupId>
+    <version>3.4.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>cdap-hbase-compat-1.2-cdh5.7.0</artifactId>
+  <name>CDAP HBase 1.2-CDH5.7.0 Compatibility</name>
+  <packaging>jar</packaging>
+
+  <repositories>
+    <repository>
+      <id>cloudera</id>
+      <url>https://repository.cloudera.com/artifactory/cloudera-repos/</url>
+    </repository>
+  </repositories>
+
+  <dependencies>
+  <dependency>
+    <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-common</artifactId>
+    <version>${project.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>co.cask.cdap</groupId>
+    <artifactId>cdap-data-fabric</artifactId>
+    <version>${project.version}</version>
+  </dependency>
+  <dependency>
+    <groupId>co.cask.tephra</groupId>
+    <artifactId>tephra-hbase-compat-1.1</artifactId>
```

Comment:
HBase 1.2.0-cdh5.7.0 found compatible with `tephra-hbase-compat-1.1` so we did not add new compat module for it in Tephra. Corresponding Tephra PR - https://github.com/caskdata/tephra/pull/117